pkg/port/cli/client.go

```diff
@@ -107,3 +102,20 @@ func WithCreateMissingRelatedEntities(createMissingRelatedEntities bool) Option
 		pc.CreateMissingRelatedEntities = createMissingRelatedEntities
 	}
 }
+
+func getCachedToken(ctx context.Context, cfg *config.ApplicationConfiguration) (*oauth2.Token, error) {
```

Comment:
`getToken`? If we don't have an existing `oauth2.TokenSource` we create a new one